packages/timeline-state-resolver/src/devices/vizMSE.ts

```diff
@@ -1183,6 +1235,11 @@ class VizMSEManager extends EventEmitter {
 	private _filterEnginesToClear(engines: Engine[], channels: string[] | 'all'): Array<{ host: string; port: number }> {
 		return engines.filter((engine) => channels === 'all' || (engine.channel && channels.includes(engine.channel)))
 	}
+
+	public async setConcept(cmd: VizMSECommandSetConcept): Promise<void> {
+		await this._rundown?.setAlternativeConcept(cmd.concept)
```

Comment:
This should await `this.getRundown()`, otherwise it might fail to set the concept in some cases (e.g gateway restart)
Also some debug logging like in other commands would be nice.

Reply:
Updated